spyder/plugins/updatemanager/widgets/status.py

```diff
@@ -61,8 +58,8 @@ class UpdateManagerStatus(StatusBarWidget):
 
     def __init__(self, parent):
 
-        self.tooltip = self.BASE_TOOLTIP
-        super().__init__(parent, show_spinner=True)
+        self.tooltip = None
```

Comment:
```suggestion
        self.tooltip = ""
```

Avoid possible issues when setting the tooltip because Qt expects a string.